infra/modules/gcp-psoxy-bulk/main.tf

```diff
@@ -163,7 +163,7 @@ resource "google_cloudfunctions_function" "function" {
   project     = var.project_id
   region      = var.region
 
-  available_memory_mb   = var.available_memory_mb
+  available_memory_mb   = coalesce(var.available_memory_mb, 1024)
```

Comment:
Just asking: can be this null here? in the vars file, the default is already set up as 1024.

Reply:
it seemed to revert to `256` in that case